### PR TITLE
[Task-803]-Change the description field as rich text editor in Kanban Board with collapse

### PR DESCRIPTION
**Pull request by rajiravenantech**
Change the description field as rich text editor in Kanban Board with collapse

### CODE REVIEW

**Review by shanmugapriyaravenantech — CHANGES_REQUESTED** [review]
In files upload only image files can be uploaded. Need to upload other format files too

File: src/app/components/AdvancedKanbanBoard/EditTask.tsx [hunk 1]
```diff
@@ -261,7 +311,14 @@ const EditTask: React.FC<EditTaskProps> = ({
                     onClick={toggleContentEditMode}
                     style={{ padding: "10px 0" }}
                   >
-                    <Typography variant="body1">{localTask.content}</Typography>
+                    <Typography>
+                      {/* Sanitize and display content using dompurify */}
+                      <div
+                        dangerouslySetInnerHTML={{
```
Comment (karthicravenantech) [inline src/app/components/AdvancedKanbanBoard/EditTask.tsx:317]: Need to remove the dangerouslySetInnerHTML

**Comment by karthicravenantech** [comment]
In comment text editior need to upload other format files like csv,pdf and txt, but its working only images

**Review by karthicravenantech — CHANGES_REQUESTED** [review]
Need to remove the dangerouslySetInnerHTML

**Pull request merged**